python/main.py

```diff
@@ -23,8 +24,25 @@ def root():
     return {"message": "Hello, world!"}
 
 @app.post("/items")
-def add_item(name: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...)):
     logger.info(f"Receive item: {name}")
+
+    # define format of the dictionaty to be stored
+    data = {"item":[]}
```

Comment:
I think this variable doesn't seem to be used anywhere.
Could you let me know what you want to do on this line?

Reply:
I tried to store new data in the "data". So it seems that I should change from "file_data" to "data" in the script